ia/src/game.py

```
@@ -0,0 +1,45 @@
+import process
```

Ruff (0.11.9)

`process` imported but unused

Remove unused import: `process`

(F401)